templates/main.html.tera

```diff
@@ -71,7 +71,9 @@
             </div>
             {% if config.SHOW_METADATA %}
                 <div class="loupe-metadata loupe-metadata-bottom invisible">
-                    <div class="loupe-metadata-element hidden"><i class="fa-solid fa-image"></i> <span class="loupe-metadata-filename fw"></span></div>
+                    {% if config.SHOW_FILENAME_IN_METADATA %}
+                        <div class="loupe-metadata-element hidden"><i class="fa-solid fa-image"></i> <span class="loupe-metadata-filename fw"></span></div>
```

Comment:
`fa-file-image` instead of `fa-image` would help visually distinguish from the "title" field. But maybe this is still too similar and `fa-file` would be better?
https://fontawesome.com/icons/file-image?s=solid&f=classic
https://fontawesome.com/icons/file?s=solid&f=classic